package.json

```diff
@@ -24,13 +24,16 @@
     "@trpc/client": "^10.16.0",
     "@trpc/react-query": "^10.16.0",
     "@trpc/server": "^10.16.0",
+    "@types/mockery": "^1.4.30",
     "axios": "^1.3.2",
     "dotenv": "16.0.3",
     "kysely": "^0.23.4",
+    "mockery": "^2.1.0",
     "ms": "^2.1.3",
     "next": "13.1.6",
     "next-auth": "^4.20.1",
     "nodemailer": "^6.9.2",
+    "nodemailer-mock": "^2.0.1",
```

Comment:
let's move these to `devDependencies` since they're not needed to run the application